src/main/kotlin/org/rust/lang/core/psi/RsPsiFactory.kt

```diff
@@ -477,7 +460,7 @@ class RsPsiFactory(
             ?: error("Failed to create trait type")
 }
 
-private fun RsFunction.getSignatureText(subst: Substitution): String? {
+fun RsFunction.getSignatureText(subst: Substitution): String? {
```

Comment:
Looks like it's worth to move this function in the same place as `createMembers `